bilingual-module/README

```diff
@@ -37,10 +37,11 @@ $ make
 # make install
 ```
 
-You can use `./autogen.sh` instead of `./configure` in case you're compiling from
-source.
+You can use `./autogen.sh` instead of `./configure` in case you're compiling
+from source.
 If you installed any prerequisite language packages using a `--prefix`
```

Comment:
I intended for you to combine these sentences on the same line. There's no reason each sentence needs to start on its own line.

Reply:
> That is, it doesn't render as a link:

GitHub doesn't render links with the IRC protocol:

```md
[irc://irc.freenode.net/#apertium](irc://irc.freenode.net/#apertium)
```

is rendered as: [irc://irc.freenode.net/#apertium](irc://irc.freenode.net/#apertium)